src/LaraCart.php

```diff
@@ -483,16 +486,20 @@ public function total($format = true, $withDiscount = true)
      *
      * @param boolean $format
      * @param boolean $withDiscount
+     * @param boolean $withTax
      *
      * @return string
      */
-    public function subTotal($format = true, $withDiscount = true)
+    public function subTotal($format = true, $withDiscount = true, $withTax = false)
```

Comment:
In my use case, I have :
- total cart price without taxes
- taxes total
- total cart price with taxes